### PR TITLE
[DOC] add comments about expired keys content as s null-array

### PR DESCRIPTION
**Pull request by ashtul**
MOD-3515

### CODE REVIEW

File: docs/commands/ft.search.md [hunk 1]
```diff
@@ -68,8 +68,9 @@ Searches the index with a textual query, returning either documents or just ids.
 
 - **SORTBY {attribute} [ASC|DESC]**: If specified, the results are ordered by the value of this attribute. This applies to both text and numeric attributes. Attributes needed for **SORTBY** should be declared as **SORTABLE** in the index, in order to be available with very low latency (notice this adds memory overhead)
 
-- **LIMIT first num**: Limit the results to
-  the offset and number of results given. Note that the offset is zero-indexed. The default is 0 10, which returns 10 items starting from the first result.
+- **LIMIT first num**: Limit the results to the offset and number of results given.
+  Note that the offset is zero-indexed. The default is 0 10, which returns 10 items starting from the first result.
+  If a result key expires during the query, its content will be a null-array.
```
Comment (oshadmi) [inline docs/commands/ft.search.md:73]: Should also be aligned with the "Expiration of hashes during a search query" below?
It says

> name and content of the hash will not be returned.

Should it be something like:
name and content of the key will return as null

File: docs/commands/ft.aggregate.md [hunk 2]
```diff
@@ -54,6 +54,7 @@ Runs a search query on an index, and performs aggregate transformations on the r
 * **LIMIT {offset} {num}**. Limit the number of results to return just `num` results starting at index
   `offset` (zero-based). AS mentioned above, it is much more efficient to use `SORTBY … MAX` if you
   are interested in just limiting the output of a sort operation.
+  If a key expires during the query, an attempt to `load` its content, will return a null-array.
```
Comment (rrelledge) [inline docs/commands/ft.aggregate.md:57]:
```suggestion
  If a key expires during the query, an attempt to `load` the key's value will return a null array. 
```

File: docs/commands/ft.search.md (hunk 1 above)
Comment (rrelledge) [inline docs/commands/ft.search.md:73]:
```suggestion
  If a result key expires during the query, its content will be a null array.
```

File: docs/commands/ft.search.md [hunk 3]
```diff
@@ -88,7 +89,7 @@ Searches the index with a textual query, returning either documents or just ids.
 If **NOCONTENT** was given, we return an array where the first element is the total number of results, and the rest of the members are document ids.
 
 {{% alert title="Expiration of hashes during a search query" color="info" %}}
-If a hash expiry time is reached after the start of the query process, the hash will be counted in the total number of results but name and content of the hash will not be returned.
+If a hash expiry time is reached after the start of the query process, the hash will be counted in the total number of results but name and content of the key will return as null.
```
Comment (rrelledge) [inline docs/commands/ft.search.md:92]:
```suggestion
If a hash expires after the query process starts, the hash will be counted in the total number of results, but the key's name and content will return as null.
```

**Review by rrelledge — COMMENTED** [review]
Looks good to me. Here's a few small suggestions to consider.

**Review by rafie — APPROVED** [review]

**Pull request merged**